snapshot_manager/testing_farm/tfutil.py

```diff
@@ -0,0 +1,411 @@
+"""TBD"""
```

Comment:
Should this be removed?

Reply:
Fixed in 449cba921e034d0304dbc13f07f1c7bc4d426fb8